test/e2e/test/command/command.go

```diff
@@ -21,12 +22,20 @@ type Command struct {
 }
 
 // Execute runs the command and returns the output.
-func (c *Command) Execute(ctx context.Context) ([]byte, error) {
+func (c *Command) Execute(ctx context.Context) ([]byte, []byte, error) {
 	cmd := exec.CommandContext(ctx, c.executable, c.args...) //nolint:gosec
 	cmd.Dir = c.workDir
 	cmd.Env = append(os.Environ(), c.env...)
 
-	return cmd.CombinedOutput()
+	var stderr bytes.Buffer
+	cmd.Stderr = &stderr
```

Comment:
To be sure I understand correctly, this will fix #5668 because the message `Kubeconfig user entry is using deprecated API version client.authentication.k8s.io/v1alpha1. Run 'aws eks update-kubeconfig' to update.` is written to `stderr`?

Reply:
I would say that it's more a workaround than a fix, but yes that's the idea. We would only try to unmarshal `stdout`, not the various warnings we may have in `stderr`

Reply:
Yes, that was the idea.